apps/api/v2/src/ee/calendars/controllers/calendars.controller.ts

```diff
@@ -1,23 +1,24 @@
 import { CalendarBusyTimesInput } from "@/ee/calendars/inputs/calendar-busy-times.input";
-import { CalendarsService } from "@/ee/calendars/services/calendars";
+import { CalendarsService } from "@/ee/calendars/services/calendars.service";
 import { GetUser } from "@/modules/auth/decorators/get-user/get-user.decorator";
 import { AccessTokenGuard } from "@/modules/auth/guards/access-token/access-token.guard";
 import { Controller, Get, Logger, UseGuards, Body } from "@nestjs/common";
 import { User } from "@prisma/client";
 
 import { SUCCESS_STATUS } from "@calcom/platform-constants";
+import { Calendars } from "@calcom/platform-libraries";
 import { ApiResponse } from "@calcom/platform-types";
 import { EventBusyDate } from "@calcom/types/Calendar";
 
 @Controller({
-  path: "ee/overlay-calendars",
+  path: "ee/calendars",
   version: "2",
 })
 @UseGuards(AccessTokenGuard)
-export class CalendarController {
+export class CalendarsController {
   private readonly logger = new Logger("ee overlay calendars controller");
 
-  constructor(private readonly overlayCalendarsService: CalendarsService) {}
+  constructor(private readonly calendarsService: CalendarsService) {}
 
   @Get("/busy-times")
   async getBusyTimes(
```

Comment:
can we put @Query instead of @Body in this pr already

Reply:
Had it in a different branch, but pushed now in commit [body -> query](https://github.com/calcom/cal.com/pull/13559/commits/d2a64081c35511e793ff72e2dd9997f9e68f1082)